content/en/design/guides/designing-interviews.md

```diff
@@ -73,6 +75,60 @@ At the end of the interview, you wrap up in a way that makes participants feel a
 
 Things you should do include:
 
-- Asking if there is anything participants would like to add.
-- Telling your participants what you are going to do with their data and what the value is for them.
-- Thanking them for taking the time to help with your research.
+- asking if there is anything participants would like to add,
+- telling your participants what you are going to do with their data and what the value is for them, and
+- thanking them for taking the time to help with your research.
+
+<br clear="all">
+
+ *****
+
+### Example Interview Guide
+
+#### Introduction
+
+_You have been identified as someone with significant expertise and experience, and we have asked you to participate in this interview. Your responses are confidential. We may need to record the session for use later on as we compile notes from this interview. The conversation will take 30 to 45 minutes of your time._
+
+##### CHW Supervisors
+
+###### Questions on supervision
+
+1. _What does your typical day look like?_
+2. _How many CHWs have you been assigned to supervise?_
+3. _What are your main supervisory responsibilities?_
+4. _Which tools do you use during supervision of CHWs?_
+5. _Which are the health areas that CHWs are expected to report?_
+6. _How often do you interact/meet with CHWs for official work assignments?_
+7. _Do you shadow CHWs during their visits to households or other field activities? How often?_
+8. _What informs the mode or approach of supervision?_
+9. _What supervision activities are involved?_
+
+###### Questions on reporting
+
+1. _How often do you collect data from CHW?_
+2. _What data is usually or typically hard to collect accurately for CHWs?_
+3. _What are commonest challenges observed or reported by CHWs during meetings?_
+4. _How often do CHWs submit their reports?_
+5. _What challenges have you experienced while compiling and reporting the data from CHWs?_
+6. _What improvements do you think would help improve the reporting process?_
+
+#### Ending the interview
+
+_We have come to the end of the interview. Do you have anything to add that you feel we have not covered?_
+
+_Thank you so much for taking the time to speak with me today. Learning about how you conduct supervision and reporting activities will really help us understand how to create solutions to support your work. If you think of anything else or you have any questions, you are more than welcome to get in touch. I also want to ask if we can contact you again if we think of other questions or if something is unclear. Is that alright?_
+
+{{% alert title="Observation checklist" %}}
```

Comment:
We can remove this section on observation as it is a separate user research method

Reply:
I've removed the observation checklist.